capstone/src/main/java/com/google/sps/Objects/Option.java

```diff
@@ -51,6 +54,26 @@ public void addVote(String userId) {
    * @return list of user ids
    */
   public List<String> getVotes() {
-    return votes;
+    return this.votes;
+  }
+
+  public long getId() {
+    return this.id;
+  }
+
+  public static Option fromEntity(Entity entity) {
+    long id = entity.getKey().getId();
+    String text = (String) entity.getProperty("text");
+    List<String> votes = (ArrayList<String>) entity.getProperty("votes");
+    return new Option(id, text, votes);
+  }
+
+  public Entity toEntity() {
```

Comment:
Didn't mean to force you to make this an instance method, but I personally think this is cleaner this way :)